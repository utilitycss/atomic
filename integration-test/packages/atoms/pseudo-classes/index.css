.hover {
  color: green;
}

.hover:hover {
  color: red;
}

.focus {
  color: green;
}

.focus:focus {
  color: red;
}

.button {
  background-color: black;
}

.button:disabled {
  background-color: grey;
}

.hover-electron {
  electrons: (black, black_hover);
}
