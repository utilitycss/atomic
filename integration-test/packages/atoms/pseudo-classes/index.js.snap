module.exports = {
  hover: "l-ITF2",
  focus: "x7QluR",
  button: "gPy6Qt",
  hoverElectron: "YcXvtG DJ9tNH"
};